### PR TITLE
bitwarden.md

### PR DESCRIPTION
**Pull request by salvationtower**
## Purpose

Bitwarden document advising that container will install as root, however user can change to what ever user is required after install.

## Approach

advising user how to chagne the permissions if required

#### Open Questions and Pre-Merge TODOs

- [ ] Use github checklists. When solved, check the box and explain the answer.

#### Learning

_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

## Requirements

- [ ] These changes meet the standards for [contributing](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CONTRIBUTING.md).
- [ ] I have read the [code of conduct](https://github.com/GhostWriters/DockSTARTer/blob/master/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by salvationtower** [comment]
File to go here - https://github.com/GhostWriters/DockSTARTer/tree/docs/docs
and have link navigation link here - https://github.com/GhostWriters/DockSTARTer/blob/docs/mkdocs.yml

**Comment by salvationtower** [comment]
This addresses #666

File: docs/bitwarden.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+layout: default
+---
+
+Bitwarden Install
+
+When installing the Bitwarden container, the installer will install under Appdata directory as the root user, however once it is installed you can change the owner/group of it to whatever is required
+
+Run the below command (from a terminal) to change the permissions if required. 
+
+```(sudo) chown -R owner:group ~/.docker/appdata/bitwarden```
```
Comment (nemchik) [inline docs/bitwarden.md]: Can we change this to:
\`sudo chown -R owner:group ~/.config/appdata/bitwarden\`

Note the single \` (tripple is used for multi-line code blocks) and the corrected path.

**Comment by salvationtower** [comment]
ok have updated the code

**Comment by nemchik** [comment]
Merged! Thanks for this!

**Pull request merged**